src/server.js

```diff
@@ -312,7 +314,7 @@ function initializeClient(socket, client, generateToken, token) {
 				type: "notification",
 				timestamp: Date.now(),
 				title: "The Lounge",
-				body: "🚀 Push notifications have been enabled"
+				body: " Push notifications have been enabled"
```

Comment:
why this change?

Reply:
Because of some copy pasting that skipped it, I guess. Not intended.